reader/views.py

```diff
@@ -1743,6 +1743,9 @@ def visualize_links_through_rashi(request):
     json_file = "../static/files/torah_rashi_torah.json" if level == 1 else "../static/files/tanach_rashi_tanach.json"
     return render(request,'visualize_links_through_rashi.html', {"json_file": json_file})
 
+def talmudic_relationships(request):
+    json_file = "../static/files/talmudic_relationships_data.json"
```

Comment:
This data is available from our data store. 
Best practice would be to load it from there.  

See the classes in `sefaria/model/person.py` and the views `reader.views.person_index()` and `reader.views.person_page()`.